dist/tools/teensy-loader-cli/build.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env sh
+
+#
+# Copyright (c) 2015 Nick v. IJzendoorn <nijzendoorn@engineering-spirit.nl>
+#
+# A simple script to fetch and build the teensy_loader_cli tool used for the 
+# Teensy boards.
+# For more information see: https://www.pjrc.com/teensy/loader_cli.html
+#
+
+# get sources from repository
+git clone https://github.com/PaulStoffregen/teensy_loader_cli.git
+cd teensy_loader_cli
+
+# resolv build host in a hacky way
+UNAME=`uname`
+TARGET="WINDOWS";
+if [ "x$UNAME" = "xLinux" ]; then
+    TARGET="LINUX";
+elif [ "x$UNAME" = "xDarwin" ]; then
+    TARGET="MACOSX";
+fi;
+
+# check if the library exists
+if [ "x$TARGET" != "xWINDOWS" ]; then
+    if ! ldconfig -p | grep 'libusb' > /dev/null; then
```

Comment:
I'm sorry, a typo sneaked into my comment above. Please change to
`pkg-config --list-all | grep 'libusb'`